filters.txt

```diff
@@ -465,6 +467,7 @@ ronnie.cz###fixedMenu
 ronnie.cz###rek3
 ronnie.cz###rodkaz
 rouming.cz##.roumingLista
+rozbor-dila.cz,rozbor-dila.eu,rozborknihy.cz,studijni-svet.cz,ekonomie-ucetnictvi.cz,biologie-chemie.cz##aside[id*="ai_widget-"]
```

Comment:
how about using `##.ai_widget` instead?

Reply:
Not possible, there is a number in id and every page has a different number
![obrazek](https://github.com/tomasko126/easylistczechandslovak/assets/149894969/09100081-c99e-468f-aa20-d05b77094ef7)
![obrazek](https://github.com/tomasko126/easylistczechandslovak/assets/149894969/6b65e3a0-8cc9-4ba4-b43e-1cf9a5d31626)

Reply:
Did'nt know this is possible, yes this works too